scripts/check_if_pr_is_low_risk.py

```diff
@@ -264,9 +264,6 @@ def check_if_pr_is_changelog_pr(pr, diff_files, file_diffs):
         str. An empty string if the PR is a changelog PR and low-risk,
         else a message explaining why the PR is not low-risk.
     """
-    source_repo = pr['head']['repo']['full_name']
-    if source_repo != 'oppia/oppia':
-        return 'Source repo is not oppia/oppia'
```

Comment:
Instead of removing this check should we replace it with the correct value? [I believe there must be some filed in `pr` which will return `oppia/oppia`, right? can we use that?]

Reply:
Hm, I thought changelog PRs are always made from the release coordinator's forked repo. Is this not the case?

Reply:
We had tried to make it more generic, I'm fine with assuming what you have suggest above as well, can you please update the docstring if we are assuming changelog PRs are always made from the release coordinator's forked repo?

Reply:
Sure, done!